### PR TITLE
Disable do_pings

### LINKED ISSUE #462: Disable do_pings by default
Something like:

```
add_filter( 'schedule_event', function( $event ) {
    if ( 'do_pings' === $event->hook ) {
        return false;
    }
    return $event;
} );
```

With the option to enable, if preferred.

### PR DESCRIPTION
**Pull request by joshbetz**
Fixes #462

### CODE REVIEW

**Comment by ethitter** [comment]
Should we remove the `VIP_CUSTOM_PINGS` stuff along with this change? https://github.com/Automattic/vip-go-mu-plugins/blob/0b4ad12862a2aef65982eadc94fd96521c7acf01/misc.php#L85

File: performance/do-pings.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+// Disable pings by default
+function disable_pings( $event ) {
+	if ( 'do_pings' === $event->hook ) {
+		return false;
+	}
+
+	return $event;
+}
+add_action( 'schedule_event', 'disable_pings' );
```
Comment (ethitter) [inline performance/do-pings.php]: Core needs the namespace specified as well.

**Comment by joshbetz** [comment]
Partially deployed in r89565-deploy. Will remove `VIP_CUSTOM_PINGS` code later today.

**Comment by mjangda** [comment]
@joshbetz looks like `VIP_CUSTOM_PINGS` still needs to be removed from the `deploy` repo.

**Comment by joshbetz** [comment]
Thanks for the reminder. Deployed in r90185-deploy.

**Pull request merged**